packages/stellar-axelar-std-derive/src/upgradable.rs

```diff
@@ -42,47 +21,44 @@ pub fn upgradable(name: &Ident, args: MigrationArgs) -> TokenStream2 {
             }
         }
 
+        type __MigrationData = <#name as stellar_axelar_std::interfaces::CustomMigratableInterface>::MigrationData;
```

Comment:
Inline this? If someone defines two upgradable contracts in the same scope (e.g. a unit test), then the typedef will clash